src/main/java/tools/jackson/dataformat/xml/deser/FromXmlParser.java

```diff
@@ -35,10 +35,10 @@ public class FromXmlParser
     implements ElementWrappable
 {
     /**
-     * The default name placeholder for XML text segments is empty
-     * String ("").
+     * The default name placeholder for XML text segments is &lt;xml:text&gt;
+     * @since 3.0.0 Is now &lt;xml:text&gt; - was empty String ("") before.
```

Comment:
nit: 3.0.0 -> 3.0 (we don't usually use patch version for API level changes)

Reply:
fixed